src/optimizer/filter_combiner.cpp

```diff
@@ -239,6 +239,9 @@ FilterCombiner::GenerateTableScanFilters(std::function<void(unique_ptr<Expressio
 				auto &column_ref = (BoundColumnRefExpression &)*func.children[0].get();
 				auto &constant_value_expr = (BoundConstantExpression &)*func.children[1].get();
 				string like_string = constant_value_expr.value.str_value;
+				if (like_string.size() == 0) {
```

Comment:
Makes sense :-), I think .empty() is a bit cleaner though

Reply:
However, shouldn't this case be optimized away?

Reply:
> However, shouldn't this case be optimized away?

Indeed it should. But the check still needs to be there.